.ci/travis/travis.sh

```diff
@@ -265,10 +265,8 @@ no-error-test-sbe)
   ;;
 
 no-error-xwiki)
-  # non-snapshot version until https://github.com/checkstyle/checkstyle/issues/9236
-  # CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
-  #                   --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
-  CS_POM_VERSION=8.39
+   CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
```

Comment:
restore indentation (remove single space).

we do not have Check for this, but indentation matter.

Reply:
Sorry again, mind jumbled up!
Corrected.